Core/include/Acts/Visualization/IVisualization3D.hpp

```diff
@@ -72,6 +72,12 @@ class IVisualization3D {
   /// Remove all contents of this helper
   ///
   virtual void clear() = 0;
+
+  virtual ~IVisualization3D() = default;
+
+  /// Start a new object context
+  /// @param name The name of the object
+  virtual void object(const std::string& name) = 0;
```

Comment:
Hmm could we find a more descriptive name like `newNamedContext(...)` or so?

Reply:
I was going off of the naming in OBJ, where it's called `object`. I can change the name if you want, but until we have a second (real) format (PLY isn't really used much) I would keep the level of abstraction somewhat close to OBJ. What do you think?

Reply:
fine for me